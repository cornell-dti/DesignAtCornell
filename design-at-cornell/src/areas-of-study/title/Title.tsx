import React from 'react';
import { TitleBackground, QuizButton } from '../AreasOfStudyStyles';

const Title = () => (
  <TitleBackground>
    <h1>Areas of Study</h1>
    <p>A guide to the different paths and stories you may pave in your design career at Cornell</p>
    <QuizButton>
      <p>Take the Majors & Minors Quiz!</p>
    </QuizButton>
  </TitleBackground>
);

export default Title;
