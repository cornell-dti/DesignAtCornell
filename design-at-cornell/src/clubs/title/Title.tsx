import React from 'react';
import FilterBar, { Props } from './FilterBar';
import { TitleBackground, TitleContainer, QuizButton } from '../../components/TitleStyles';
import { colors } from '../../constants/colors';

const Title = (props: Props) => (
  <TitleBackground color={colors.cyan}>
    <TitleContainer>
      <h1>Student Organizations</h1>
      <p>Find your design people, your creative community :)</p>
    </TitleContainer>
    <FilterBar {...props} />
  </TitleBackground>
);

export default Title;
