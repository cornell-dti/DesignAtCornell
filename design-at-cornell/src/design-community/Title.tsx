import React from 'react';
import { TitleBackgroundImage, TitleContainerBox } from '../components/TitleStyles';
import background from '../static/images/designcommunity-background.svg';

const Title = () => (
  <TitleBackgroundImage style={{ backgroundImage: `url(${background})` }}>
    <TitleContainerBox>
      <h1>Welcome to your design community!</h1>
      <p>
        Grow relationships with like-minded individuals by connecting with the amazing design
        faculty at Cornell! Expand your knowledge of design through exploring relevant articles.
      </p>
    </TitleContainerBox>
  </TitleBackgroundImage>
);

export default Title;
