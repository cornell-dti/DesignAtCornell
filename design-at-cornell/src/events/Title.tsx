import React from 'react';
import { TitleBackground, TitleContainer } from '../components/TitleStyles';
import { colors } from '../constants/colors';

const Title = () => (
  <TitleBackground color={colors.blue}>
    <TitleContainer>
      <h1>Events</h1>
      <p>Learn, grow and share with your creative community.</p>
    </TitleContainer>
  </TitleBackground>
);

export default Title;
