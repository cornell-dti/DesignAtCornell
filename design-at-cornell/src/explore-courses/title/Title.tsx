import React from 'react';
import FilterBar from './FilterBar';
import { TitleBackground, TitleContainer, QuizButton } from '../ExploreCoursesStyles';
import { TitleProps } from '../types/PropertyTypes';

const Title = (props: TitleProps) => (
  <TitleBackground>
    <TitleContainer>
      <h1>Explore Courses</h1>
      <p>
        Your central hub for finding and learning about courses to enrich your design skillset and
        broaden your horizons.
      </p>
      <QuizButton>
        <p>Taking the Design Course Quiz</p>
      </QuizButton>
    </TitleContainer>
    <FilterBar {...props} />
  </TitleBackground>
);

export default Title;
