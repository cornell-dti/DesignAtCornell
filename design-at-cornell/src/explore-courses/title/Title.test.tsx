import React from 'react';
import renderer from 'react-test-renderer';
import Title from './Title';
import 'jest-styled-components';

describe('<Title />', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<Title />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
