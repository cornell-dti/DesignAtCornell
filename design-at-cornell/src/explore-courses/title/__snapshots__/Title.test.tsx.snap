// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title /> renders correctly 1`] = `
<div
  className="sc-gsTCUz hosgWk"
>
  <div
    className="sc-dlfnbm gzNRPu"
  >
    <div
      className="sc-eCssSg jtbzzI"
    >
      <h1>
        Explore Courses
      </h1>
      <p>
        Your central hub for finding and learning about courses to enrich your design skillset and broaden your horizons.
      </p>
    </div>
    <div
      className="sc-jSgupP fwkluo"
    >
      <div
        className="sc-gKsewC RUKyW"
      >
        <p>
          Design Areas
        </p>
        <img
          alt="expand"
          src="down-arrow.png"
        />
      </div>
      <div
        className="sc-gKsewC RUKyW"
      >
        <p>
          Major/Minors
        </p>
        <img
          alt="expand"
          src="down-arrow.png"
        />
      </div>
      <div
        className="sc-gKsewC RUKyW"
      >
        <p>
          Semester
        </p>
        <img
          alt="expand"
          src="down-arrow.png"
        />
      </div>
      <div
        className="sc-gKsewC RUKyW"
      >
        <p>
          Level
        </p>
        <img
          alt="expand"
          src="down-arrow.png"
        />
      </div>
      <div
        className="sc-gKsewC RUKyW"
      >
        <p>
          Credits
        </p>
        <img
          alt="expand"
          src="down-arrow.png"
        />
      </div>
      <input
        placeholder="Name, Keywords, Topics, Etc"
        type="text"
      />
    </div>
  </div>
  <div
    className="sc-hKgILt fQZyOa"
  >
    <div
      className="sc-jSgupP fwkluo"
    >
      <p>
        Take the Design Course Quiz!
      </p>
    </div>
  </div>
</div>
`;
